arbeitszeit/email_notifications.py

```diff
@@ -40,6 +40,9 @@ class WorkerInvitation:
 @dataclass
 class WorkerRemovalNotification:
     worker_email: str
+    worker_name: str
+    worker_id: str
```

Comment:
We are here still in the business logic and should use `UUID`s and not strings, where possible. No need to convert the worker_id to string.